packages/ecmascript-runtime-client/versions.js

```diff
@@ -9,7 +9,7 @@ setMinimumBrowserVersions({
   // (pre-Edge) from the modern classification. #9818 #9839
   ie: 12,
   firefox: 45,
-  mobile_safari: 10,
+  mobileSafari: 10,
```

Comment:
Nice!

Reply:
The funny/sad thing is that Meteor is actually responsible for the camel-casing [here](https://github.com/meteor/meteor/blob/4e59d8bb6514c718960b69a46abda591b3390c19/packages/webapp/webapp_server.js#L116). 🤦‍♂️